tools/bmh_test/run_local_bmh_test_setup.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+set -eux
+
+REPO_ROOT=$(realpath "$(dirname "${BASH_SOURCE[0]}")/../..")
+
+cd "${REPO_ROOT}" || exit 1
+
+# Check if the script is run as root
+if [[ $EUID != 0 ]]; then
+    echo "ERROR: This script must be run as root" 
+    exit 1
+fi
+
+# Install virtinst, libvirt clients and daemon system
+apt-get -y update
+apt-get -y install virtinst libvirt-clients libvirt-daemon-system
```

Comment:
I would rather check that virt-install is available end error out if not. So instead of trying to install these things, explain to the developer what is needed and let them set it up as they want.
We can also skip the root requirement then.